packages/taxdat/R/stan_input_helpers.R

```diff
@@ -2183,3 +2183,85 @@ get_loctime_combs_mappings <- function(stan_data) {
   
   stan_data
 }
+
+
+#' get_adm0_od_param
+#'
+#' @param sf_cases_resized 
+#'
+#' @return
+#' @export
+#'
+#' @examples
+get_adm0_od_param <- function(sf_cases_resized,
+                              res_time,
+                              cases_column) {
+  
+  # Get single-year data at adm0
+  ts_subset <- sf_cases_resized %>% 
+    get_admin_level_data(res_time = res_time,
+                         admin_levels = 0,
+                         censorings = "full")
+  
+  # Get the maximum of adm0 observations
+  max_adm0_obs <- max(ts_subset[[cases_column]])
+  
+  if (max_adm0_obs > 1e4) {
+    od_param <- 1e3
+  } else {
+    od_param <- 1e2
+  }
+  
+  od_param
+} 
+
+
+#' drop_obs_by_OC
+#'
+#' @param sf_cases_resized 
+#' @param model_time_slices 
+#' @param res_time 
+#'
+#' @return
+#' @export
+#'
+#' @examples
+#' 
+drop_obs_by_OC <- function(sf_cases_resized,
+                           res_time) {
+  
+  # Add obs id for filtering
+  sf_cases_resized <- sf_cases_resized %>% 
+    dplyr::mutate(tmp_obs_id = dplyr::row_number())
+  
+  # Get single-year data at adm0
+  ts_subset <- sf_cases_resized %>% 
+    get_admin_level_data(res_time = res_time,
+                         admin_levels = 0,
+                         censorings = NULL)
+  
+  # Get maximum tfrac obs by OC
+  max_tfrac_obs <- ts_subset %>% 
```

Comment:
@javierps Don't we need a condition that checks whether the max tfrac exceeds the config$censoring_thresh?

Reply:
Correct, this is now implemented using the censoring column of sf_cases_resized